e2e/cypress/e2e/issue_workflow_test/Traceability/issueTransformationEvent.cy.ts

```diff
@@ -0,0 +1,42 @@
+import IssuePage from 'cypress/page/issuePage';
+
+class TransformationEventIssueFlow extends IssuePage {
+  testGenerateDPPWorkflow() {
+    this.generateWorkflow(
+      'Orchard Facility',
+      'Issue Traceability Event',
+      'DigitalTraceabilityEvent',
+      'DigitalTraceabilityEvent-v0.5.0.json',
```

Comment:
Should we be paramaterising the version here, or are we happy to be updating it. I see you've already paramaterised it below, why not here?

Reply:
It's not clear to me how you resolved this either - that is, I see here that the version is included in the 'DigitalTraceabilityEvent-v0.5.0.json' arg, and in the new version you're instead passing the constant DIGITAL_TRACEABILITY_EVENT which is 'DigitalTraceabilityEvent', ie. without the version. I assume it's now also using the parameter below, passed in to `runUntpTest`.

Again, resolving this to unblock. Also I'm not sure if the 4000 line diff in `app-config.json` is intended for review, but I've not looked through it. Maybe next time if you could split mechanical changes into separate branch where it is clear it's not for review but a mechanical change (it looks like a lot of it is formatting, though I see there are some changes).